src/Description/Description.js

```diff
@@ -1,11 +1,14 @@
 import React from "react";
 import "./Description.css";
 import { usePlayerState } from "../playerState";
+import { useEncounterState } from "../encounterState";
 
 const Description = () => {
   const player = usePlayerState();
+  const { npc } = useEncounterState();
   // starting string, can be changed according to player's last location
-  let description = "You are in a grassy field. ";
+  let description =
+    npc !== null ? `You see a ${npc.name}. ` : "You are in a grassy field. ";
```

Comment:
Two questions, first should a npc have a description that we can insert here or is this going to be wear insert the first part dialogue path?

Reply:
I think having an NPC description would be good! It'd certainly make for more dynamic dialogue than this.